Istio migration docs (#4246)
kyma-project/kyma: docs/application-connector/08-02-get-client-certificate.md

## Call the Application Registry and Event services on local deployment

> - Kyma installation on Minikube by default uses self signed certificate so it is required to skip TLS verification:

Reviewer: ```suggestion
- Since Kyma installation on Minikube uses the self-signed certificate by default, skip TLS verification:
```

> > **NOTE:** Adding `NodePort` to the the Gateway URL is no longer required.

Reviewer: ```suggestion
> **NOTE:** You no longer need to add `NodePort` to the the Gateway URL.
```
